libgnucash/engine/qofbook.cpp

```diff
@@ -107,7 +107,7 @@ qof_book_option_num_autoreadonly_changed_cb (GObject *gobject,
 #define PARAM_NAME_NUM_FIELD_SOURCE "split-action-num-field"
 #define PARAM_NAME_NUM_AUTOREAD_ONLY "autoreadonly-days"
 
-QOF_GOBJECT_GET_TYPE(QofBook, qof_book, QOF_TYPE_INSTANCE, {});
+G_DEFINE_TYPE_WITH_CODE(QofBook, qof_book, QOF_TYPE_INSTANCE, {});
```

Comment:
If there's no code shouldn't it be ```G_DEFINE_TYPE```?

Reply:
good question

Reply:
Compiles fine, makes more sense. I'll use that.